onnx/backend/test/case/node/rmsnormalization.py

```diff
@@ -0,0 +1,172 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+from __future__ import annotations
+
+import numpy as np
+
+import onnx
+from onnx.backend.test.case.base import Base
+from onnx.backend.test.case.node import expect
+
+
+# RMS normalization's reference implementation
+def _rms_normalization(X, W, axis=-1, epsilon=1e-5):  # type: ignore
```

Comment:
Include this in the reference runtime?

Reply:
If this is a complete implementation (instead of just for a special subset), yes, it would make sense to move it to the reference runtime and use it (well, have a shared implementation, whereever the code resides).